src/reader/ArticleReader.js

```diff
@@ -285,69 +325,72 @@ export default function ArticleReader({ api, teacherArticleID }) {
         UMR_SOURCE={UMR_SOURCE}
         articleProgress={scrollPosition}
       />
-      <h1>
-        <TranslatableText
-          interactiveText={interactiveTitle}
-          translating={translateInReader}
-          pronouncing={pronounceInReader}
-          setIsRendered={setReaderReady}
-        />
-      </h1>
-      <div
-        style={{
-          marginTop: "1em",
-          marginBottom: "2em",
-          display: "flex",
-          flexDirection: "row",
-          justifyContent: "space-between",
-        }}
-      >
-        <ArticleAuthors articleInfo={articleInfo} />
-        <div style={{ display: "flex", flexDirection: "row" }}>
-          <ArticleSource url={articleInfo.url} />
-          <ReportBroken
-            api={api}
-            UMR_SOURCE={UMR_SOURCE}
-            history={history}
-            articleID={articleID}
+      <div id="text">
+        <h1>
+          <TranslatableText
+            interactiveText={interactiveTitle}
+            translating={translateInReader}
+            pronouncing={pronounceInReader}
+            setIsRendered={setReaderReady}
           />
+        </h1>
+        <div
+          style={{
```

Comment:
isn't it more readable if we define this div elsewhere, maybe in a `sc.js` file? And give it a semantic? And separate the CSS from the code?